src/test/java/com/team1/dealerApp/video/movie/MovieServiceTest.java

```diff
@@ -0,0 +1,187 @@
+package com.team1.dealerApp.video.movie;
+
+
+import com.team1.dealerApp.video.Genre;
+import com.team1.dealerApp.video.VideoStatus;
+import org.apache.coyote.BadRequestException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.time.Year;
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class MovieServiceTest {
+
+    // Simula il MovieRepository
+    @Mock
+    private MovieRepository movieRepository;
+
+
+    private final MovieMapper movieMapper = new MovieMapper();
```

Comment:
Potresti mettere movie mapper in cima dato che si tratta di un'istanza e lasciare tutti i mock uniti tra loro